tests/static-checks.sh

```diff
@@ -242,6 +242,32 @@ need_chronic() {
 		"Usually it is distributed with the 'moreutils' package of your Linux distribution."
 }
 
+# Mark a section begin.
+#
+# If running within a Github Action then it will start grouping log lines, else
+# just print the section title.
+#
+# Parameters:
+#	$*: the section title
+#
+section_begin() {
+	if [ -n "${GITHUB_ACTION:-}" ];then
+		echo "::group::${*}"
```

Comment:
Do you think it's worth distinguishing between GH action and local execution? The `::group::NAME` format is quite readable by humans as well (after they observe enough logs).

Note I don't have anything against the functions (besides I'd call them `log_group` or `log_section` to emphasize it's about logs), but I don't think we should change the output based on GITHUB_ACTION variable.